packages/compiler/src/lib/paging.ts

```diff
@@ -187,6 +187,17 @@ type PagingPropertyKind =
 
 export interface PagingProperty {
   readonly property: ModelProperty;
+  /**
+   * If the paging property is nested, this will contain the path to the paging property in the model
+   * and array length will be greater than one. 
+   * 
+   * You can use this to generate the path to the property in the model with following code:
+   * @example
+   ```ts
+   const path = paing.output.pageItems.path.map((prop) => prop.name).join(".");
+   ```
```

Comment:
```suggestion
   * ```ts
   * const path = paing.output.pageItems.path.map((prop) => prop.name).join(".");
   * ```
```